coralproject/talk
# Banned word list

## What does this PR do?

Adds the ability to define banned words in the admin panel. Automatically rejects comments which include these words and notifies the user.

## How do I test this PR?

1. Go to `http://localhost:3000/admin/configure`
2. Click 'Banned words list`
3. Enter one or more comma separated words. "Save" button should turn green.
4. Click "Save Changes"
5. Navigate to `http://localhost:3000`
6. Enter a comment with a banned word.
7. You should receive a notification informing you that your comment includes a banned word.
8. Go to `http://localhost:3000/admin`
9. You should see you comment under the "rejected" tab.

## Note

A few spanish translations are missing in `/client/coral-admin/src/translations.json` and in `/client/coral-framework/translations.json`. Search for the phrase `¡traduceme!` if you'd like to help out.
